README.md

```diff
@@ -29,25 +28,45 @@
 
 </div>
 
-
 # Installation 👨‍💻
 
 ### Linux 🐧
-<a href="https://appimage.github.io/Upscayl/">
-<img src="https://user-images.githubusercontent.com/25067102/191270389-9de37c0f-39a8-41f1-a659-8dd4e7b8ac28.png" width="250px"/>
-</a>
 
-<a href="https://aur.archlinux.org/packages/upscayl-bin">
-<img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" width="250px"/>
-</a>
+<div align='center'>
+  <a href="https://appimage.github.io/Upscayl/">
+    <img src="https://user-images.githubusercontent.com/25067102/191270389-9de37c0f-39a8-41f1-a659-8dd4e7b8ac28.png" width="250px"/>
+  </a>
+
+  <a href='https://flathub.org/apps/org.upscayl.Upscayl'>
+    <img src='https://dl.flathub.org/assets/badges/flathub-badge-en.png' alt='Download on Flathub' Height="70px" maxWidth='250px'/>
+  </a>
+
+  <a href="https://aur.archlinux.org/packages/upscayl-bin">
+    <img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" width="250px"/>
+  </a>
+</div>
+
+#### [Flatpak](https://flatpak.org)
+
+1. Go to [Flathub/Upscayl](https://flathub.org/apps/org.upscayl.Upscayl)
+
+2. Press the "Install" button. This will download the `.flatpakref` file.
+
+3. Execute the downloaded `.flatpakref` file. This will open your disrtibution's software center.
+
+4. Press the installation button in your disrtibution's software center.
+
+#### [AppImage](https://appimage.org/)
 
 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
 
 2. Download the `.AppImage` file.
 
 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
 
-You can also download the flatpak version and double click the flatpak file to install via Store but wait for the full release, we'll be pushing it to Flathub for easy access.
+*Please note:*
+- *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*
+- *The AppImage needs to be [made executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80) after download.*
```

Comment:
Should these notes be in the individual sections instead? Like the first one under flatpak and the second one under AppImage? Also, I don't think we should link the titles, as these are all notable enough and no other heading is linked.

Reply:
[Done](https://github.com/Danik1601/upscayl/commit/c38162f86f66a43c067a30b72a1a1ade57888391)